src/window-info.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Provides information about the screen dimensions.
+ */
+export class windowInfo {
+  /**
+   * Retrieves the width of the screen.
+   *
+   * @returns The width of the screen in pixels, or undefined if not available.
+   */
+  static getScreenWidth(): number | undefined {
+    return window.screen.width;
```

Comment:
`window.screen.width` and `screen.width` will produce the same result. Both properties access the width property of the global screen object.

though, when using only `screen.width` we get the following eslint problem `// eslint-disable-next-line no-restricted-globals`